app/src/main/java/com/sapuseven/untis/workers/AutoMuteSetupWorker.kt

```diff
@@ -94,6 +103,11 @@ class AutoMuteSetupWorker(context: Context, params: WorkerParameters) :
 							if (item.periodData.isCancelled() && !automuteCancelledLessons)
 								return@forEach // lesson is cancelled
 
+
+							if (hiddenSubjects.contains(item.periodData.subjects.first()))
```

Comment:
Same case as in `NotificationSetupWorker`
```suggestion
							if (item.periodData.subjects.isNotEmpty() && hiddenSubjects.containsAll(item.periodData.subjects))
```